src/main/scala/com/databricks/spark/csv/CsvRelation.scala

```diff
@@ -145,10 +146,13 @@ case class CsvRelation protected[spark] (
       }
       if (this.inferCsvSchema) {
         InferSchema(tokenRdd(header), header)
-      } else{
+      } else {
```

Comment:
I think it will be cool if user could override inferred types with this feature. Let's merge the types provided manually with inferred schema.